packages/frontend/src/features/common/components/layout/userPanel/UserData.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { FontSize } from "../../../styles/constans/FontSize"
+
+const Wrapper = styled.div`
+    position: relative;
+    display: flex;
+`
+const UserPhoto = styled.img`
+    height: 100px;
+    width: 100px;
+    border-radius: 50%;
+    margin-right: 20px;
+`
+const TextInfoSection = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    margin-right: 20px;
+`
+
+const UserInfoTxt = styled.p<{ email?: any, name?: any }>`
+    padding: 4px;
+    font-size: ${props => props.name ? FontSize.MEDIUM : FontSize.VERY_SMALL};
+    font-weight: ${props => props.email ? "" : "600"};
+`
+
+const UserEmail = styled(UserInfoTxt)`
+    font-size: ${FontSize.SMALL};
+`
+const LogoutBtn = styled.button`
+    display: flex;
+    position: absolute;
```

Comment:
czemu absolute?

Reply:
patrzyłem na widok mobilki nie zauważyłem tego dla frontu i tam środek przycisku był ustawiony  na lini przecięcia z dolną pozycją zdjęcia moim zdaniem dobrze wygląda jak tak jest ustawiony

Reply:
ale co to ma wspólnego z absolute? (nie ma potrzeby użycia tu absolute). Czemu nie ustawiłeś tego marginesem lub inaczej?